File: scala_app/tests/HelloTest.scala
package tests
import org.junit.Test
import org.junit.Assert._

/* Ensure our greetings are polite */
class HelloTest {
  @Test
  def helloTest() = {
      val value=System.getProperty("name");
      assertEquals("name5511", value);

      val value2=System.getProperty("name2");
      assertEquals("name23", value2);


  }
}

File: java_app/tests/HelloTest.java
package tests;
import org.junit.Test;
import static org.junit.Assert.*;
/* Ensure our greetings are polite */
public class HelloTest {
  @Test
  public void helloTest() {
      String value=System.getProperty("name");
      assertEquals("name5511", value);

      String value2=System.getProperty("name2");
      assertEquals("name23", value2);
  }
}
